### ISSUE TITLE
DoE module v1.0.3 is not compatible with jamovi 2.3.13

### ISSUE THREAD

**Issue by indycool79** [UBC-LFS/jamovi-modules#2]
Hello,

I tried to install the module but it is not compatible and therefore refuse to launch. Could you please make it work? :)

**Comment by exponentian** [comment]
Hi indycool79,

The DoE v1.0.3 module is now compatible with jamovi (2.2.5 solid) which is recommended for most users. Please try to install jamovi v2.2.5 on your machine.

Or, do you have any reasons to use v2.3.13?

Thanks.

**Comment by indycool79** [comment]
yes, the 2.3 is translated into my language and it is easier for my course to have it this way (If the DoE module could be made an "official" module, I could translate it too).
And the 2.3 have new features (i.e. identification of outliers for example) and fewer bugs.
